contrib/kind.sh

```diff
@@ -25,8 +25,10 @@ run_kubectl() {
 
 usage()
 {
+    #     12345678901234567890123456789012345678901234567890123456789012345678901234567890
```

Comment:
what's this :) ?

Reply:
Limiting the helptext for CLI to 80 characters. I can remove if it bugs you.

Reply:
Done.